content/02.intro.md

```diff
@@ -36,30 +37,38 @@ Easy-to-use software packages have brought the techniques of the field out of th
+A recent book from Goodfellow et al. covers neural network architectures in detail [@url:http://www.deeplearningbook.org/].
+
 | Term | Definition |
 |----------------------------|-------------------------------------------------------------------------------------------|
-| Neural network  (NN) | Machine-learning approach where inputs are fed into one or more hidden layers, producing an outer layer |
-| Deep learning (DL) approach | NN with multiple hidden layers |
 | Supervised learning | Machine-learning approaches with goal of prediction of labels or outcomes |
 | Unsupervised learning | Machine-learning approaches with goal of data summarization or pattern identification |
-| Convolutional neural network (CNN) | NN used for grid data -- such as images or equally-spaced time points - that considers convolutions instead of linear transformations, leading to increased sparsity and thus improved efficiency |
+| Neural network  (NN) | Machine-learning approach inspired by biological neurons where inputs are fed into one or more layers, producing an output layer |
+| Deep neural network | NN with multiple hidden layers. Training happens over the network, and consequently such architectures allow for feature construction to occur alongside optimization of the overall training objective. |
 | Feed-forward neural network (FFNN) | NN that does not have cycles between nodes in the same layer |
 | Multi-layer perceptron (MLP) | Type of FFNN with at least one hidden layer where each deeper layer is a nonlinear function of each earlier layer |
-| Recurrent neural network (RNN) | NN used for sequential data -- such as time series or genomic data -- by using cycles between nodes in the hidden layers, in contrast to FFNN |
-| Long short-term memory (LSTM) model | Special type of RNN that can learn longer-term dependencies |
-| Autoencoder (AE) | NN that sets the outer layer to be similar to the input layer, used for example in dimension reduction |
-| Variational Autoencoder (VAE) | AE with an added constraint of learning normally distributed features |
-| Generative neural network | NN approach that uses models trained to generate data similar to the collected data, leading to smaller number of parameters |
-| Generative adversarial network (GAN) | Generative NN approach that uses two networks, one that generates samples from training data and one that discriminates between generated and training data |
-| Adversarial training | Constructing artificial training examples that are maliciously designed to fool a neural network in order to make it robust to such attacks (no relation to GANs) |
+| Convolutional neural network (CNN) | A NN with layers in which connectivity preserves local structure. These are used for grid data -- such as images or equally-spaced time points. Such networks have fewer parameters, increased sparsity and thus improved efficiency, and performance is often maintained _if the data meet the underlying assumptions_. |
```

Comment:
Should we say `sequence or grid` instead of `grid`?  The second sentence may need some rephrasing or punctuation, I wasn't sure what was part of a list.